.teamcity/settings.kts

```diff
@@ -270,7 +268,6 @@ class TestBuild(val os: String, val arch: String, val version: String, buildId:
             "386", "amd64" -> equals("teamcity.agent.jvm.os.arch", if (os == "mac") "x86_64" else "amd64")
             "arm64" -> equals("teamcity.agent.jvm.os.arch", "aarch64")
             "ppc64le" -> equals("teamcity.agent.jvm.os.arch", "ppc64le")
-            "riscv64" -> equals("teamcity.agent.jvm.os.arch", "riscv64")
```

Comment:
Let's comment this out instead with a note mentioning we're in need of a builder. May inspire some help in the future.

Reply:
Done.